modules/promotions/module.php

```diff
@@ -98,6 +99,8 @@ public function enqueue_react_data(): void {
 				'react',
 				'react-dom',
 				'backbone-marionette',
+				'elementor-editor-modules',
```

Comment:
The promotions pop up wasn't working when all experiments were deactivated, because of missing dependency scripts. In this way, it always works.